### PR TITLE
relecture

### PR DESCRIPTION
**Pull request by ThomasFaria**
- ligne 155 : "The accuracy on automatically-coded descriptions; "
  -> en vrai c'est pas exactement ça, c'est l'accuracy sur les auto codé + supposition que tout ceux en dessous du seuil sont bien codé. Ca biaise un peu à la hausse si on estime que les annotateurs se trompe. Peut etre que je devrais changer ca dans le dashboard)

- ligne 171: The updated training set would be an extraction where relevant observations have been identified and their code modified.
->Ici dire que dans notre cas pas un probleme car on retrain from scratch et que c'est rapide ? (En fait t'en parle dans le paragraphe suivant donc à ignorer)

- ligne 186 : @fig-accuracy-training-size displays the accuracies on the Label Studio evaluation set from models trained on SIRENE 4 data as well as subsets from SIRENE 3 data (as well as training set size) as functions of the earliest year included in SIRENE 3 training data
-> J'avais pas le graphique sous les yeux au moment ou j'ai lu donc j'avais pas trop compris mais avec le graphique je vois ce que tu veux dire. Je pense quand meme qu'il faudrait un peu simplifier cette phrase si possible

 - ligne 207 : This is consistent with the idea that CamemBERT models language better and should therefore have a higher classification accuracy on longer, more complex descriptions.
-> Manque un mot la non ?

Sinon d'accord avec tout ce que tu y écris. J'aime bien l'idée que les gros modèles s'ils méritent pas qu'on les mettent en prod car coûts >>> gains ils peuvent etre utile pour optimiser l'annotation

### CODE REVIEW

**Comment by tomseimandi** [comment]
@ThomasFaria pour la remarque 1 c'est vrai aussi pour les plots avec top-k accuracies par exemple ?

**Comment by tomseimandi** [comment]
merci bcp !!! pour la dernière remarque "models" c'est le verbe la phrase passe je pense
